modules/box/src/lib.rs

```diff
@@ -29,6 +29,9 @@ static mut A: [u64; ARGBUF_LEN / 8] = [0; ARGBUF_LEN / 8];
 #[no_mangle]
 static AL: i32 = ARGBUF_LEN as i32;
 
+#[no_mangle]
+static SELF_ID: [u8; MODULE_ID_BYTES] = [0u8; MODULE_ID_BYTES];
```

Comment:
I would make this a SELF_ID: ModuleId rather than an opaque buffer, is there something speaking against this?